tensorboard/backend/application.py

```diff
@@ -546,3 +554,11 @@ def _clean_path(path):
     if path != "/" and path.endswith("/"):
         return path[:-1]
     return path
+
+
+def _placeholder_assets_zip_provider():
+    """Defines a default assets zip provider containing a dummy index.html."""
+    memfile = io.BytesIO()
```

Comment:
OK—this target has `srcs_version = "PY2AND3"`, but it looks to me like
the semantics of `io.BytesIO` are unchanged from 2 to 3, so seems fine.

Reply:
Which target? `:application` is only "PY3" now:

https://github.com/tensorflow/tensorboard/blob/f22060fb97423178ff2a67a363715a5dbb6735c4/tensorboard/backend/BUILD#L57

Reply:
Submitting but let me know if there's something I should follow up on here.